apps/api_web/lib/api_web/controllers/prediction_controller.ex

```diff
@@ -275,6 +291,16 @@ defmodule ApiWeb.PredictionController do
     |> select(:stop_id)
   end
 
+  defp all_stops_and_route_patterns(stop_ids, route_pattern_ids, matchers) do
+    trip_ids =
+      %{}
+      |> Map.put(:route_patterns, route_pattern_ids)
```

Comment:
Could be just `Map.new(route_patterns: route_pattern_ids)` or simply `%{route_patterns: route_pattern_ids}` I think